tutorials/actors_demo/actors_demo.md

```diff
@@ -0,0 +1,169 @@
+
+Let's define the trajectory as a sequence of waypoints:
+
+```xml
+        <trajectory id="0" type="walk">
+            <waypoint>
+                <time>0</time>
+                <pose>0 0 1.0 0 0 0</pose>
+            </waypoint>
+            <waypoint>
+                <time>2</time>
+                <pose>2.0 0 1.0 0 0 0</pose>
+            </waypoint>
+            <waypoint>
+                <time>2.5</time>
+                <pose>2 0 1.0 0 0 1.57</pose>
+            </waypoint>
+            <waypoint>
+                <time>4</time>
+                <pose>2 2 1.0 0 0 1.57</pose>
+            </waypoint>
+            <waypoint>
+                <time>4.5</time>
+                <pose>2 2 1.0 0 0 3.142</pose>
+            </waypoint>
+            <waypoint>
+                <time>6</time>
+                <pose>0 2 1 0 0 3.142</pose>
+            </waypoint>
+            <waypoint>
+                <time>6.5</time>
+                <pose>0 2 1 0 0 -1.57</pose>
+            </waypoint>
+            <waypoint>
+                <time>8</time>
+                <pose>0 0 1.0 0 0 -1.57</pose>
+            </waypoint>
+            <waypoint>
+                <time>8.5</time>
+                <pose>0 0 1.0 0 0 0</pose>
+            </waypoint>
+        </trajectory>
+    </script>
+</actor>
+```
+
+Within the `<trajectory>` tag we can define a series of waypoints which our actor will follow. The `<trajectory>` has two attributes: `id` and `type`. The `type` should have the same name as the animation `walk`. The `trajectory` parameters are as follow:
+
+* `waypoint`: There can be any number of waypoints in a trajectory. Each waypoint consists of a time and a pose:
+    * `time`: The time in seconds, counted from the beginning of the script, when the pose should be reached.
+    * `pose`: The pose which should be reached.
+
+Run the world and you should see our actor moving in a square following the waypoints.
+
+**Notes**:
+
+* The order in which waypoints are defined is not important, they will follow the given times.
+* The trajectory is smoothed as a whole. This means that you'll get a fluid motion, but the exact poses contained in the waypoints might not be reached.
+
+Now it's your turn! Try out different trajectories description.
+
+## TODO
+
+* adding actors to our environment (fuel)
+* move the actor with script
+* closed loop control using plugin
+* remove the sensor plugin from the sdf?
+* use an already built world and add to it or start new?
+* comment blocks in sdf
+* GIF for skeleton and trajectory movement and combined
+* when link to code link to the raw version.
+* Make the movement reasonable
+* refer to the final sdf of the tutorial
+* make the move smoother if possible
+* I think trajectory motion is not supported. but I can make it with the (talk_b.dae)
```

Comment:
Can you elaborate on what this one means? `talk_b.dae` is not the kind of animation that you want to add a trajectory to.

Reply:
I mean having the actor all actor's links moving as one group along trajectory (no skeleton movement)
I tried with the `talk_b.dae` and it follow the trajectory

Reply:
Oh got it. How about explaining that in the **Scripted trajectory** section? You can explain that animations that have displacement on the X axis, like `walk.dae`, will have the skeleton animated while following a trajectory. But for animations that don't, such as `talk_b.dae`, their skeletons won't move if there's a trajectory.